modules/__tests__/describePopState.js

```diff
@@ -14,11 +14,17 @@ const describePopState = (createHistory) => {
     })
 
     it('is called when browser navigation is used', (done) => {
-      unlisten = history.listenBefore(() => {
-        done()
-      })
+      setTimeout(() => {
```

Comment:
Here is the issue - if this outer setTimeout is not in place, the spy will actually be called twice. I'm unsure if this is because of `window.location.hash = ''` on HashHistory-test.js:L19, or `history.push('/home')` on L8 of this file, but it seems likely to be the former since it interferes with the following hashchange test I added, as well. In any case, some way to bypass those first events is necessary for this test (such as this brute-force timeout), otherwise it's trivially testing that setup.